content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,99 @@
+# Persisting Experiments
+
+When you run an experiment with `dvc exp run`, it produces temporary objects in
+the Git repository. These are listed with `dvc exp list` and `dvc exp show`.
+
+Although the experiments as the temporary, they can be shared with
+`dvc exp push` and retrieved from other Git repositories with `dvc exp pull`. In
+this section we'll see how to bring them to the standard Git workflow with
+`dvc exp apply` and `dvc exp branch`.
```

Comment:
This one sentence is enough intro for the doc I think:

```suggestion
Let's see how to bring `dvc experiments` to the standard Git workflow with
`dvc exp apply` and `dvc exp branch`.
```

Reply:
Or maybe add some more motivation as to why you'd want to bring exps to a regular git flow.

Reply:
I added an initial sentence and introduced the commands.